code/game/objects/items/food/packaged.dm

```diff
@@ -0,0 +1,234 @@
+// Pre-packaged meals, canned, wrapped, and vended
+
+// Cans
+/obj/item/food/canned
```

Comment:
Why is this a food wtf it's literally just canned drink with food reagents

Reply:
Food code is very ancient in a lot of places 😔